sdk/include/psdk/pathcch.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Michael Müller
```

Comment:
This header seems to duplicate https://github.com/reactos/reactos/blob/master/sdk/lib/pathcch/pathcch.h